analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -75,9 +75,9 @@ private const val GRADLE_VERSION = "5.6.4"
 private const val PUB_LOCK_FILE = "pubspec.lock"
 
 private val flutterCommand = if (Os.isWindows) "flutter.bat" else "flutter"
-private val pubCommand = if (Os.isWindows) "pub.bat" else "pub"
+private val pubCommand = if (Os.isWindows) "pub.bat" else "dart pub"
```

Comment:
Please explain in the commit message why this change is necessary as part of the update.

Also, for single-file changes our convention is to use the name of the file as the topic, and start with a capitalized letter after the colon: "Pub: Upgrade Flutter to version 2.0.6"

Finally, a link to the change log of Flutter 2.0.6 would be nice for completeness.